### DISCUSSION TITLE
Considering using uv - An extremely fast Python package installer and resolver written in Rust

### DISCUSSION THREAD

**Discussion by BigSamu** [fastapi/full-stack-fastapi-template#1316] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
N/A
```

### Description

Have you consider using [uv](https://astral.sh/blog/uv) instead of poetry in [full-stack-fastapi-template](https://github.com/fastapi/full-stack-fastapi-template/issues) boilerplate? Benchmarks show that uv is extremely fast compared to poetry (screenshot attached below)

![image](https://github.com/user-attachments/assets/ff1b55e9-0673-4d19-a2d0-e02c41a24616)

### Operating System

Windows

### Operating System Details

N/A

### Python Version

N/A

### Additional Context

This question is related to a suggestion for opening an issue in [full-stack-fastapi-template](https://github.com/fastapi/full-stack-fastapi-template/issues) repo. Waiting approval from @tiangolo to do this.

**Comment by alejsdev** [comment, marked as answer]
Hi @BigSamu, thanks for the interest in the project :v: 
We started working on that since the day they released uv for projects. We'll continue once they support --system (it's already on their plans after bug fixes) :smile: :rocket:

**Reply by BigSamu** [reply]
Awesome!!! Thanks @alejsdev !!!
